src/background.c

```diff
@@ -751,7 +751,7 @@ handle_request_background_in_thread_func (GTask *task,
         subtitle = g_strdup_printf (_("%s requests to be started automatically and run in the background."), info ? g_app_info_get_display_name (info) : id);
       else
         subtitle = g_strdup_printf (_("%s requests to run in the background."), info ? g_app_info_get_display_name (info) : id);
-      body = g_strdup (_("The ‘run in background’ permission can be changed at any time from the application settings."));
+      body = g_strdup (_("The ‘run in background’ permission can be changed at any time from the app settings."));
```

Comment:
is this referring to the settings for the app, or to Applications settings found in Settings apps?

Reply:
Application settings found in Settings app. But what about other DEs on this and "app" name?

Reply:
If this is the case, then it would be good to have this be "applications settings" since it would be referring to general settings.

Reply:
Is this worse than what is it right now? Should we just go ahead and merge this?

At the end of the day, this is generic code shared by all environments so being more specific seems out of the question.